### DISCUSSION TITLE
how do I get rid of github workflow

### DISCUSSION THREAD

**Discussion by ALBUE** [community/community#25117] [Actions]
<p>I am just learning GitHub.  I added somehow .github/workflow.   How do I get rid of it?  It gives me errors of some kind.</p>

**Comment by BrightRan** [comment, marked as answer]
<p>If you do not want to run any workflow for you repository, you can directly delete all the YAML files under _ <strong>.github/workflow</strong> _ from your repository.</p>
<p>You also can disable GitHub Actions feature on repository level and Organization level.</p>
<p>To disable Actions on a specific repository, on the <strong>Settings</strong> tab of this repository, navigate to <strong>Actions</strong> , select <strong>Disable Actions for this repository</strong>.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/9ae688d7fe6c73bf66ff01cb15a03458c7f5e55b.png" data-download-href="/uploads/short-url/m6jrNMLjSgs0OSzKkuOTM6bzcbF.png?dl=1" title="disable_actions_repo.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181088996-b2859056-08f8-49d6-b2e2-3d00eb60bfc2.png" alt="disable_actions_repo.PNG" data-base62-sha1="m6jrNMLjSgs0OSzKkuOTM6bzcbF" width="690" height="290" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/9ae688d7fe6c73bf66ff01cb15a03458c7f5e55b_2_690x290.png, https://aws1.discourse-cdn.com/github/original/1X/9ae688d7fe6c73bf66ff01cb15a03458c7f5e55b.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/9ae688d7fe6c73bf66ff01cb15a03458c7f5e55b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/9ae688d7fe6c73bf66ff01cb15a03458c7f5e55b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">disable_actions_repo.PNG</span><span class="informations">1008×424 27.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>To disable Actions on a specific Organization, navigate to <strong>Organization settings</strong> &gt; <strong>Member privileges</strong> , select <strong>Disable Actions for the organization</strong> , this will disable Actions on all repositories in this Organization.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/95535dd8a81fb8aeaed4879f5273d872485a773a.png" data-download-href="/uploads/short-url/liZKOW0zSMGLMZyGA3ShcWjamIq.png?dl=1" title="disable_actions_org.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181088997-9736135c-0356-455f-8bd0-88025dec492d.png" alt="disable_actions_org.PNG" data-base62-sha1="liZKOW0zSMGLMZyGA3ShcWjamIq" width="690" height="201" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/95535dd8a81fb8aeaed4879f5273d872485a773a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">disable_actions_org.PNG</span><span class="informations">1008×294 22.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If you have some troubles during using Actions, you can share more details about the problems, such as the complete error logs/messages returned to you, and the related YAML files for defining the workflows. </p>

**Comment by ALBUE** [comment]
<p>Thank you, BrightRan.  Much appreciated.  Happy Thanksgiving to you and your loved ones.</p>

**Comment by karstengresch** [comment]
<p>Thanks for this solution on a per-repo level. Having 300+ repos which are all set to enabled Actions automatically (without my consent), I wonder if there’s a possibility to opt-out completely, on account level?</p>

**Comment by aaltotsky** [comment]
<p>Disable Action just remove the Action button. You can add “/actions” to the repo url to see the existing Action list. </p>
